readme.md

```diff
@@ -20,45 +20,32 @@ Here is an [example project](https://github.com/dereke/web-testing) that demonst
 
 # example
 
-```js
-import createBrowser from 'browser-monkey/create';
-
-describe('admin', function () {
-  let browser = createBrowser(document.body);
-
-  // describes the admin panel, with a search box, results and a user editor
-  let adminPanel = browser.component({
-    searchUsers() {
-      return this.find('.search');
-    },
-    userResult(name) {
-      // find a user in the results by their name
-      return this.find('.results .user', {text: name});
-    },
-    userEditor() {
-      // return the user editor, scoped to the .user-editor div.
-      return this.find('.user-editor').component({
-        name()  { return this.find('.name'); },
-        email() { return this.find('.email'); },
-        save()  { return this.find('.save'); }
-      });
-    }
+```js codesandbox: basic-example
+import createMonkey from "browser-monkey/create";
+import createTestDiv from "browser-monkey/lib/createTestDiv";
+import hyperdom from "hyperdom";
+import App from "./app";
+
+describe("beer app", () => {
+  let page;
+
+  beforeEach(() => {
+    const $testContainer = createTestDiv();
+    hyperdom.append($testContainer, new App());
+    page = createMonkey($testContainer);
```

Comment:
You could simplify these three lines to be:
```
const hyperdomMonkey = require('browser-monkey/hyperdom')
const monkey = hyperdomMonkey(new YourHyperdomApp())
```

But maybe showing the generic way helps explain how it works?

Reply:
Yes, that. I wanted to show generic usage.